docs/src/contribute/design/next-tedge-api/case-studies/rentable-equipment.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Rentable Equipment
+tags: [Documentation, MQTT, CaseStudy]
+sidebar_position: 2
+---
+
+# Rentable Equipment
```

Comment:
It's nice to have a full-fledged use-case. However, it would be good to start with a simpler one, more typical ;-)

Reply:
Yes, that is planned, but I wanted to validate the usefulness of the new design with a more complex case.

Reply:
The use-cases have been postponed to another PR as to not hold up the design. Also the use-cases will require some doc layout changes to enable presenting them in a different navbar group.